assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -166,12 +108,17 @@ export const TelemetryOptInBanner = forwardRef(
             }}
             ref={checkboxRef}
           />
-          <LabelText aria-checked={checked}>
+          <LabelText
+            as="span"
+            aria-checked={checked}
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
```

Comment:
It looks like this should actually be `EXTRA_SMALL` (12px)
<img width="532" alt="Screen Shot 2021-01-29 at 9 09 53 AM" src="https://user-images.githubusercontent.com/10720454/106298921-c7252b00-6211-11eb-90bc-7d54454b33a8.png">